src/material-experimental/mdc-chips/chip-grid.ts

```diff
@@ -378,7 +382,13 @@ export class MatChipGrid
    * @docs-private
    */
   setDescribedByIds(ids: string[]) {
-    this._ariaDescribedby = ids.join(' ');
+    // We must keep this up to date to handle the case where ids are set
+    // before the chip input is registered.
+    this._ariaDescribedbyIds = ids;
+
+    if (this._chipInput) {
+      this._chipInput.setDescribedByIds(ids);
```

Comment:
This gets called while the form field gets initialized after content is initialized:
```
MatChipInput.setDescribedByIds (chip-input.ts)
MatChipGrid.setDescribedByIds (chip-grid.ts)
MatFormField._syncDescribedByIds (form-field.ts)
MatFormField._initializeSubscript (form-field.ts)
MatFormField.ngAfterContentInit (form-field.ts)
```

This caused a `ExpressionChangedAfterCheckedError` in an application inside Google. It's probably related to how the grid/input content is provided with an `ngIf`. @crisbeto Is it valid to wrap this in a `setTimeout` to avoid change detection issues?

Reply:
Yeah, let's try it with the timeout and we can re-evaluate if it causes even more failures because the timeouts haven't been flushed.

Reply:
Timeout didn't cause any issues, we can replace the logic with this to land the change
```
if (this._chipInput) {
      // Use a setTimeout in case this is being run during change detection
      // and the chip input has already determined its host binding for
      // aria-describedBy.
      setTimeout(() => {
        this._chipInput.setDescribedByIds(ids);
      }, 0);
    }
```

Reply:
Just a heads up that this would be best done ASAP so we don't need to re-run internal tests

Reply:
Sounds good, thanks :)

Reply:
Okay, should be set to go!

Reply:
Great, we'll start loading up approvals internally to get this in